src/Symfony/Component/Config/Tests/Builder/GeneratedConfigTest.php

```diff
@@ -128,13 +160,31 @@ private function generateConfigBuilder(string $configurationClass)
             return new $fqcn();
         }
 
-        $outputDir = sys_get_temp_dir().\DIRECTORY_SEPARATOR.uniqid('sf_config_builder', true);
-
-        // This line is helpful for debugging
-        // $outputDir = __DIR__.'/.build';
-
         $loader = (new ConfigBuilderGenerator($outputDir))->build(new $configurationClass());
 
         return $loader();
     }
+
+    private function assertDirectorySame($expected, $current)
+    {
+        $expectedFiles = [];
+        foreach (new \RecursiveIteratorIterator(new RecursiveDirectoryIterator($expected, \FilesystemIterator::SKIP_DOTS)) as $file) {
+            if ($file->isDir()) {
+                continue;
+            }
+            $expectedFiles[substr($file->getPathname(), \strlen($expected))] = $file->getPathname();
+        }
+        $currentFiles = [];
+        foreach (new \RecursiveIteratorIterator(new RecursiveDirectoryIterator($current, \FilesystemIterator::SKIP_DOTS)) as $file) {
+            if ($file->isDir()) {
+                continue;
+            }
+            $currentFiles[substr($file->getPathname(), \strlen($current))] = $file->getPathname();
+        }
+
+        $this->assertSame(array_keys($expectedFiles), array_keys($currentFiles));
+        foreach ($expectedFiles as $fileName => $filePath) {
+            $this->assertSame(file_get_contents($filePath), file_get_contents($currentFiles[$fileName]));
```

Comment:
Couldn't `assertFileEquals` be used here instead?

```suggestion
            $this->assertFileEquals($filePath, $currentFiles[$fileName]);
```

Reply:
I didn't know that method exists. Thanks :heart: